test/bits/left_shift_negative.c

```diff
@@ -0,0 +1,12 @@
+#include "smack.h"
```

Comment:
Add `_fail` to the end of the file name since that is our convention for files containing errors.